index.html

```diff
@@ -240,8 +247,10 @@
     <section class="image-section hero above-fold">
       <div class="container">
         <div class="hero-logo">
```

Comment:
you can just make this `div` the `a` tag
and make `hero-logo` `display: block`

Reply:
then you won't need the floats either

Reply:
Ahhh that's so much nicer - done 👌